### PR TITLE
Multifluid compressor

### PR DESCRIPTION
**Pull request by nabily-met**
…nducators in RefMoistAir base clases

## Goal

A multifluid compressor is developed to handle both the wet compression and high temperature limit of `RefMoistAir` medium. The compressor is composed of two compressors: the first is a `RefMoistAir` compressor to represent the moist compression at the first stages, and a `FlueGases` compressor to handle to dry high temperature compression.

## Type of change 

- [ ] Bugfix
- [x] New feature
- [ ] Refactoring change
- [ ] Release & Version Update (don't forget to change the version number in `package.mo`)

Will it break anything in previous models ?

- [ ] Breaking change (If yes, make sure to point it out in the changelog)
- [x] Non-Breaking change

## Checklist

- [x] I have added the appropriate tags, reviewers, projects (and detailed the size and priority of my PR) and linked issues to this PR
- [x] I have performed a self-review of my own code
- [x] I have checked that all existing tests pass
- [ ] I have checked that my work is compatible with [OpenModelica](https://openmodelica.org/)
- [x] I have added/updated tests that prove my development works and does not break anything.
- [ ] I have made corresponding changes or additions to the documentation (in [Notion documentation](https://www.notion.so/metroscope/Metroscope-Modeling-Library-Documentation-MML3-WIP-50c8703c294446059d3b4a70d6ae4a71))
- [ ] I have added corresponding entries to the [Changelog](../CHANGELOG.md)
- [x] I have checked for conflicts with target branch, and merged/rebased in consequence

You can also fill these out after creating the PR, but make sure to **check them all before submitting your PR for review**.

### CODE REVIEW

**Review by LucileD-met — APPROVED** [review]
Keeping in mind that the de facto eta_is equality between the 2 compressors is an hypothesis like the tau/tau_moist ratio, even if it's hidden in the component.

File: MetroscopeModelingLibrary/Tests/Multifluid/Machines/AirCompressor_direct.mo [hunk 1]
```diff
@@ -0,0 +1,63 @@
+within MetroscopeModelingLibrary.Tests.Multifluid.Machines;
+model AirCompressor_direct
+  extends MetroscopeModelingLibrary.Utilities.Icons.Tests.MultifluidTestIcon;
+  import MetroscopeModelingLibrary.Utilities.Units;
+
+  package RefMoistAirMedium = MetroscopeModelingLibrary.Utilities.Media.RefMoistAirMedium;
+
+  // Boundary conditinos
```
Comment (LucileD-met) [inline MetroscopeModelingLibrary/Tests/Multifluid/Machines/AirCompressor_direct.mo:8]:
```suggestion
  // Boundary conditions
```

File: MetroscopeModelingLibrary/Tests/Multifluid/Machines/AirCompressor_reverse.mo [hunk 2]
```diff
@@ -0,0 +1,63 @@
+within MetroscopeModelingLibrary.Tests.Multifluid.Machines;
+model AirCompressor_reverse
+  extends MetroscopeModelingLibrary.Utilities.Icons.Tests.MultifluidTestIcon;
+  import MetroscopeModelingLibrary.Utilities.Units;
+
+  package RefMoistAirMedium = MetroscopeModelingLibrary.Utilities.Media.RefMoistAirMedium;
+
+  // Boundary conditinos
```
Comment (LucileD-met) [inline MetroscopeModelingLibrary/Tests/Multifluid/Machines/AirCompressor_reverse.mo:8]:
```suggestion
  // Boundary conditions
```
